libs/expo/betterangels/src/lib/screens/AddEditClient/AddEditClient.graphql

```diff
@@ -14,6 +14,22 @@ mutation UpdateClientProfile($data: UpdateClientProfileInput!) {
   }
 }
 
+mutation UpdateClientProfilePhoto($data: ClientProfilePhotoInput!) {
```

Comment:
Hi @Davit-BetterAngels 

I noticed a couple of things:

1. When I added a new client from Blue + button, I didn't see the Profile Photo section. I only got to accessed from Edit button after I created a client profile.
2. I tried to add a photo from my album or taking a picture but there is no loading icon visible. 
3. Nothing happened and loaded to the profile photo section even though I waited for a while. 

Please check. Thanks! :)

Reply:
1. We need client id to upload a photo. before creating a client we don't have an id. We are thinking of the solution for that with Mike.
2. haven't seen loading in figma. will update and push.
3. works for my device. we can look together

Reply:
@shiozakiyuko loading added

Reply:
Hey @Davit-BetterAngels 

I am able to upload the photo but I can't submit. I get an error with the snickerbar. 
I haven't tested this with Android because of the wifi issue but this is the result from iOS.
Please check. Thanks!

<img src="https://github.com/user-attachments/assets/aeae861d-746b-4ace-aeb9-b2d493731c32" width="320" />

Reply:
I get the same error on Android